docs/learn/releases.rst

```diff
@@ -0,0 +1,93 @@
+Releases
+========
+
+.. note:: Some features of releases are still in beta. If you'd like to check them out, consider joining our early adopter program.
+
+
+What is a Release?
+------------------
+
+Releases are used by Sentry to provide you with additional context
+when determining the cause of an issue. You set a release context in
```

Comment:
> You set a release context in your SDK, and can either create a release via our API or allow Sentry to create one automatically when we receive an event with a new release version.

Ideally we have sections for "set a release context in your SDK" and "create a release via the API" that follow this (before getting into commit data). We could probably clarify this sentence a bit.